back/app/services/project_copy_service.rb

```diff
@@ -78,44 +77,31 @@ def export(
   private
 
   def yml_content_builder_layouts(shift_timestamps: 0)
-    layout_images_mapping = {}
-
-    layouts = ContentBuilder::Layout.where(content_buildable_id: @project.id).map do |layout|
-      craftjs = layout.craftjs_jsonmultiloc
-
-      craftjs.each_key do |locale|
-        craftjs[locale].each_value do |node|
-          next unless ContentBuilder::LayoutService.new.craftjs_element_of_type?(node, 'Image')
-
-          source_image_code = node['props']['dataCode']
-          new_image_code = ContentBuilder::LayoutImage.generate_code
```

Comment:
So, the copied images will have the same code as the original images. It doesn't seem right that two different objects have the same UUID, but I don't see any objective reason not to do it.

Could you please ask @sebastienhoorens if it's fine? And also, why did we need `code` if we already have UUID `id`?

Reply:
No, I think you're right! That was why we went to the trouble of inserting new codes. I forgot that!

I think this approach will mean layouts will not necessarily use the correct image, and is therefore a bad idea.

Reply:
> why did we need code if we already have UUID id?

Good question 🤔